neurolib/models/wc/model.py

```diff
@@ -0,0 +1,41 @@
+import neurolib.models.wc.loadDefaultParams as dp
+import neurolib.models.wc.timeIntegration as ti
+from neurolib.models.model import Model
+
+
+
+class WCModel(Model):
+    """
+    The  standard two-population Wilson-Cowan model
+    """
+
+    name = "wc"
+    description = "Wilson-Cowan model"
+
+    init_vars = ["xs_init", "ys_init", "x_ou", "y_ou"]
+    state_vars = ["x", "y", "x_ou", "y_ou"]
+    output_vars = ["x", "y"]
+    default_output = "x"
+    input_vars = ["x_ext", "y_ext"]
+    default_input = "x_ext"
+
+
```

Comment:
To enable BOLD rescaling:

```suggestion
    # because this is not a rate model, the input
    # to the bold model must be normalized
    normalize_bold_input = True
    normalize_bold_input_max = 50
```

Reply:
The Wilson-Cowan model is a rate model, so is this step necessary then?

Reply:
As far as I remember, the outputs in the notebook are also negative. Am I missing something?

Reply:
You're right. Fixed!

Reply:
So it's not a firing rate model? Does the WC model have a transfer function at all?